sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -8195,7 +8229,7 @@ pi_result piextUSMDeviceAlloc(void **ResultPtr, pi_context Context,
     if (It == Context->DeviceMemAllocContexts.end())
       return PI_ERROR_INVALID_VALUE;
 
-    *ResultPtr = It->second.allocate(Size, Alignment);
+    *ResultPtr = It->second.get()->allocate(Size, Alignment);
```

Comment:
What is the type of `It->second`? Is that `std::shared_ptr`? If so, it should have `operator->` and we don't need to call `.get()`.